mev-boost-rs/src/service.rs

```diff
@@ -69,7 +69,13 @@ impl Service {
         let context =
             if let Some(context) = context { context } else { Context::try_from(&network)? };
         let relays = relays.into_iter().map(Relay::from);
-        let clock = context.clock(None);
+        let clock = match context.clock() {
```

Comment:
I am iterating on this API but have currently settled on something like this

https://github.com/ralexstokes/mev-rs/pull/112/files#diff-9419529fa1cde062756e22bf86ecc27a2a1c3f8a3f4caebe9549e68e2d53f1edR73-R76

do you mind updating?

Reply:
Thanks, I've merged in the changes from main.